ynr/apps/popolo/migrations/0034_populate_from_ee_divisions.py

```diff
@@ -0,0 +1,136 @@
+"""
+This data migration takes a CSV* exported from EveryElection
+and uses it to match up and change the identifier for each post it finds.
+
+When it's found a post, it makes a new `PostIdentifier` model with
+the previous `slug` value, updates the slug and sets other fields like
+start and end dates, etc.
+
+* It's actually two files to deal with two cases:
+
+1. The ballot has a post that is a division of an organisation
+2. The ballot has a post that's actually an organisation, but where
+   we created a pseudo-post for it at import time (for directly elected mayors,
+   PCCs, etc)
+
+"""
+
+from django.db import migrations
+import csv
+import re
+
+
+def update_or_create_post_identifier(post):
+    identifier_label = None
+
+    if post.identifier:
+        identifier_value = post.identifier
+        if identifier_value.startswith("unit_id"):
+            identifier_label = "unit_id"
+        if identifier_value.startswith("gss"):
+            identifier_label = "gss"
+        if re.match(r"^[A-Z]{3}", identifier_value.split(":")[0]):
+            identifier_label = "dc_slug"
```

Comment:
Shouldn't this block also handle the case where the prefix is `osni_oid:`?
For example, `nia.south-down.2016-05-05` and `nia.south-down.2017-03-02` have the `official_identifier` `osni_oid:NIE-5`.